### PR TITLE
New release v4.0.0-beta.1

### PR DESCRIPTION
**Pull request by holgerd77**
Ok, have compiled down a first version of release notes 😄, this is still WIP though.

### CODE REVIEW

**Comment by holgerd77** [comment]
Note: use "view file" on the changelog for the rendered version.

**Comment by holgerd77** [comment]
@s1na Ok, this is now ready for review. 😄

**Comment by holgerd77** [comment]
@s1na Baby 👶 sleeps and I have from now on roughly an hour to do stuff. If you read this short term it would be appreciated if you can do a (somewhat generous on spelling errors or similar) review on this, then I could take the time for the release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,230 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Promisified internal usage of async opcode handlers,
+  PR [#491](https://github.com/ethereumjs/ethereumjs-vm/pull/491)
+- Promisified `runTx` internals,
+  PR [#493](https://github.com/ethereumjs/ethereumjs-vm/pull/493)
+- Promisified `runBlock` internals, restructure, reduced shared global state,
+  PR [#494](https://github.com/ethereumjs/ethereumjs-vm/pull/494)
+
+##### Version Updates
+
+- [BREAKING] Updated `ethereumjs-account` from `2.x` to `3.x`, part of
```
Comment (s1na) [inline CHANGELOG.md]: Not sure I get why this is a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,230 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+and following `beta` versions and give us some feedback!
+
+These will be the main release notes for the `v4` feature updates, subsequent
+`beta` releases and the final release will just publish the delta changes and
+point here for reference.
+
+Breaking changes in the release notes are preeceeded with `[BREAKING]`, do a
+search for an overview.
+
+The outstanding work of [@s1na](https://github.com/s1na) has to be mentioned
```
Comment (s1na) [inline CHANGELOG.md]: :heart:

File: CHANGELOG.md (hunk 1 above)
Comment (holgerd77) [reply CHANGELOG.md]: Was meant in the sense of "people have to update their dependencies", but you are right, not really something which can be considered "breaking", have removed the label.

Comment (holgerd77) [reply CHANGELOG.md]: Once you see and are fine with the update please re-approve.

**Comment by holgerd77** [comment]
Hehe. Thanks. 😄

**Comment by holgerd77** [comment]
Out and [posted](https://www.reddit.com/r/ethereum/comments/c2fdpk/ethereumjs_vm_v400beta1_release_typescript_evm/) all over the place. 😛

@s1na Thanks again for the great work!

**Pull request merged**